src/yetibot/commands/emoji.clj

```diff
@@ -0,0 +1,87 @@
+(ns yetibot.commands.emoji
+  (:require
+    [taoensso.timbre :refer [info warn error]]
+    [yetibot.core.hooks :refer [cmd-hook]]
+    [clojure.data.json :as json]
+    [clojure.core.memoize :as m])
+  (:import
+   [com.vdurmont.emoji EmojiManager]))
+
+(defn get-all-emojis
+  "get all emoji java objects"
+  []
+  (EmojiManager/getAll))
+
+(defn parse-all-emojis
+  "create sequence of emojis"
+  []
+  (map (fn [emoji] {:unicode (.getUnicode emoji)
+                    :description (.getDescription emoji)
+                    :aliases (.getAliases emoji)
+                    :tags (.getTags emoji)
+                    }) (get-all-emojis)))
+
+(defonce emojis (m/ttl parse-all-emojis :ttl/threshold 3600000))
```

Comment:
Since `EmojiManager/getAll` is an immutable value there's no need to give a `ttl`. We can just cache it forever.

So this could be simplified as:

```clojure
(defonce emojis (parse-all-emojis))
```